joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitorHelpers.scala

```diff
@@ -129,8 +129,8 @@ trait PythonAstVisitorHelpers { this: PythonAstVisitor =>
 
       targets.foreach { target =>
         val targetWithAccessChains = getTargetsWithAccessChains(target)
-        targetWithAccessChains.foreach { case (target, accessChain) =>
-          val targetNode = convert(target)
+        targetWithAccessChains.foreach { case (trgt, accessChain) =>
```

Comment:
why not `target`? isn't that more readable that `trgt`?

Reply:
Because it shadows `target` from a view lines above.